internal/guest/runtime/runc/container.go

```diff
@@ -389,6 +413,22 @@ func (c *container) startProcess(
 		}
 	}
 
+	isLoggingSideCarContainerStr, exists := annotations["io.microsoft.bmc.logging.isLoggingSideCarContainer"]
```

Comment:
Please add a comment above this code block explaining that this is an experimental feature and has not been stabilized yet.

Reply:
added comment at the beginning of the section